src/JABWrapper/jab_wrapper.py

```diff
@@ -1719,6 +1746,45 @@ def get_visible_children(self, context: JavaObject, start_index: int) -> Visible
         if not ok:
             raise APIException('Failed to get visible children info')
         return visible_children
+    
+    def set_caret_position(self, context: JavaObject, position: int) -> bool:
+        """
+        Set the caret to the given position
+
+        Args:
+            context: the element context handle.
+            position: int representing the index where to set the caret.
+
+        Returns:
+            bool: should always be True if operation was successful.
+
+        Raises:
+            APIException: failed to call the java access bridge API with attributes or failed to make selection.
+        """
+        ok = self._wab.setCaretPosition(self._vmID, context, position)
+        if not ok:
+            raise APIException("Failed to select text")
+        return ok
+    
+    def get_caret_location(self, context: JavaObject, index: int) -> AccessibleTextRectInfo:
+        """
+        Get the coordinates of the current caret location.
+
+        Args:
+            context: the element context handle.
+            index: TODO
```

Comment:
@mmokko Do you have any idea what this index attribute might be doing? Couldn't find a documentation about it anywhere and it's also not really intuitive why there is an index argument.